pkg/api/types.go

```diff
@@ -1025,6 +1025,13 @@ const (
 	// Status code 409
 	StatusReasonConflict StatusReason = "Conflict"
 
+	// StatusReasonResourceConflict means the requested update operation cannot be completed
+	// due to a resource conflict in the operation. The client may need to alter the request.
+	// Each resource may define custom details that indicate the nature of the
```

Comment:
"Each API resource..."

This is confusing because we're using the term "resource" with 2 different meanings, but I couldn't think of a better word.